internal/provider/harness.go

```diff
@@ -175,6 +173,9 @@ func addHarnessResourceSchemaAttributes() map[string]schema.Attribute {
 			Description: "The inventory this harness belongs to. This is received as a direct input from a data.imagetest_inventory data source.",
 			Required:    true,
 			Attributes: map[string]schema.Attribute{
+				"id": schema.StringAttribute{
+					Required: true,
+				},
```

Comment:
I don't think you want these? at least these aren't being `Set` anywhere in the resource state

Reply:
Terraform requires me to have it, otherwise it chokes. It is being set: https://github.com/chainguard-dev/terraform-provider-imagetest/blob/aded2f9a35b5f02d46df0d200815ac327ccd39a5/internal/provider/inventory_data_source.go#L75-L80